test/cmdlineTests/transient_storage_layout_value_types_interleaved_with_storage/input.json

```diff
@@ -0,0 +1,13 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"fileA": {"urls": ["transient_storage_layout_value_types_interleaved_with_storage/in.sol"]}
+	},
+	"settings": {
+		"outputSelection": {
+			"fileA": {
+				"A": ["storageLayout","transientStorageLayout"]
```

Comment:
It would probably make sense to always request both in these tests (except for the ones that specifically test that `transientStorageLayout` triggers the right output), so that we can see what ends up in each one.

Reply:
Also:
```suggestion
				"A": ["storageLayout", "transientStorageLayout"]
```